bundles/mapping/mapwfs2/plugin/components/propertyArrayUtils.js

```diff
@@ -1,4 +1,5 @@
 export const WFS_ID_KEY = '_oid';
+export const WFS_FTR_ID_KEY = '__fid';
```

Comment:
We could change the server implementation to use `__fid` as the key so that we don't need two constants for the same thing.